src/assets-list/assets-list.js

```diff
@@ -0,0 +1,91 @@
+import jQuery from 'jquery';
+import template from './assets-list.pug';
+const assetUrl= 'http://fuego-app.herokuapp.com/api/assets';
+import Assets from '../services/assets';
+
+const component = jQuery('<div class="assets-list-component component"></div>');
+//console.log(component);
+
+// jQuery.getJSON(assetsURL).then(function(response) {
+// 	//console.log(response.count);
+// 	//const parameters = ;
+
+// 		const html = template({
+// 			title: 'Hola Mundo',
+// 			assets: response.result
+// 		});
+
+// 		console.log(html);
+
+// 		jQuery('#assets-app').html(html);
+// });
+
+
+jQuery.getJSON(assetUrl).then(renderAssetsHtml);
+
+function loadAssets() {
+	//Ahora en services
```

Comment:
No comments in spanish. If you are going to code in english, leave comments in english.